src/Tests/Indices/Monitoring/IndicesStats/IndicesStatsWithShardsInformationApiTests.cs

```diff
@@ -0,0 +1,84 @@
+﻿using System;
+using System.Linq;
+using Elasticsearch.Net;
+using FluentAssertions;
+using Nest;
+using Tests.Framework;
+using Tests.Framework.Integration;
+using Tests.Framework.ManagedElasticsearch.Clusters;
+
+namespace Tests.Indices.Monitoring.IndicesStats
+{
+	public class IndicesStatsWithShardsInformationApiTests : ApiIntegrationTestBase<WritableCluster, IIndicesStatsResponse,
+		IIndicesStatsRequest, IndicesStatsDescriptor, IndicesStatsRequest>
+	{
+		public IndicesStatsWithShardsInformationApiTests(WritableCluster cluster, EndpointUsage usage) : base(cluster, usage)
+		{
+		}
+
+		private class NoopClass
```

Comment:
Maybe use `RandomString()` for index name?

Reply:
Not needed :)

Reply:
using `RandomString()` or `CallUniqueValues` in `IntegrationSetup` would remove the need for a type

Reply:
Done, fixed with unique call value.